### ISSUE TITLE
Exception is thrown when fields in findings.yaml are missing

### ISSUE THREAD

**Issue by philippslowak** [smartbugs/smartbugs#147]
**Bug description**

An exception is thrown when fields (e.g. `descr_short`) are missing in `findings.yaml` of the corresponding tool when running with `--sarif` option.

Expected behavior: Missing fields are ignored

**Platform**

- SmartBugs version: 2.0.2
- Python version: 3.10.9.final.0 (64 bit)
- OS: Darwin 22.1.0
- CPU: Apple M1

**Comment by gsalzer** [comment]
@philippslowak Can you provide more information, most importantly the traceback of the exception, and maybe the input? I've checked the code, and it seems that every access to the information in `findings.yaml` handles the case that the field does not exist. I ran some experiments with various stripped down versions of Mythril's `findings.yaml` file, but were not able to trigger any exceptions.

**Comment by philippslowak** [comment]
@gsalzer Sure, the behavior was encountered while working on PR #149. The input was `./smartbugs -t sfuzz -f samples/open_address_lottery.sol --sarif`.  

Traceback:

```python
./smartbugs -t sfuzz -f samples/open_address_lottery.sol --sarif
Welcome to SmartBugs 2.0.2!
Starting task 1/1: sfuzz and samples/open_address_lottery.sol
Process SpawnProcess-1:
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.10/3.10.9/Frameworks/Python.framework/Versions/3.10/lib/python3.10/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/opt/homebrew/Cellar/python@3.10/3.10.9/Frameworks/Python.framework/Versions/3.10/lib/python3.10/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/smartbugs/sb/analysis.py", line 116, in analyser
    duration = execute(task)
  File "/smartbugs/sb/analysis.py", line 80, in execute
    sarif_result = sb.sarif.sarify(task_log["tool"], parsed_result["findings"])
  File "/smartbugs/sb/sarif.py", line 7, in sarify
    "runs": [ run_info(tool, findings) ]
  File "/smartbugs/sb/sarif.py", line 14, in run_info
    "tool":    tool_info(tool, fnames),
  File "/smartbugs/sb/sarif.py", line 22, in tool_info
    "rules": [ rule_info(tool["id"], fname) for fname in fnames ]
  File "/smartbugs/sb/sarif.py", line 22, in <listcomp>
    "rules": [ rule_info(tool["id"], fname) for fname in fnames ]
  File "/smartbugs/sb/sarif.py", line 42, in rule_info
    v = rule_shortDescription(info_finding)
  File "/smartbugs/sb/sarif.py", line 92, in rule_shortDescription
    return info_finding.get("descr_short")
AttributeError: 'NoneType' object has no attribute 'get'
Analysis completed in 0:02:02.
```

Contents of `findings.yaml`:

```yaml
Block Number Dependency:
  classification: SWC-120
  severity: Low

Dangerous Delegate Call:

Exception Disorder:

Freezing Ether:

Gasless Send:

Integer Overflow:
  classification: SWC-101
  severity: High

Integer Underflow:
  classification: SWC-101
  severity: High

Reentrancy:
  classification: SWC-107
  severity: High

Timestamp Dependency:

```

**Comment by gsalzer** [comment]
@philippslowak Thanks for the detailed report. I have fixed the issue in commit #ef0ff8f .
